lib/process-services-cloud/src/lib/form/services/form-utils.service.ts

```diff
@@ -0,0 +1,32 @@
+﻿/*!
+ * @license
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Injectable } from '@angular/core';
+import { FormModel, FormVariableModel } from '@alfresco/adf-core';
```

Comment:
Within the same library please use relative imports instead of depending the barrel files.

Reply:
This seems to be something my IDE (Rider) does automatically, I'll have a look ...